platform/server/src/test/java/org/motechproject/server/bootstrap/MessageBrokerUtilTest.java

```diff
@@ -0,0 +1,32 @@
+package org.motechproject.server.bootstrap;
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.motechproject.server.util.MessageBrokerPingUtil;
+
+
+/**
+ * Created by atish on 18/7/15.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class MessageBrokerUtilTest {
+
+    private static final String BROKER_URL = "vm://localhost?broker.persistent=false";
+
+    private static final String INCORRECT_BROKER_URL = "tcp://failServer.com?broker.persistent=false";
+
+    @Test
+    public void verifyBrokerPingSuccess(){
```

Comment:
If this expects an actual ActiveMQ it should be an IT. The unit tests build will fail on this, since we don't have activemq even installed on its slave